addons/web/static/lib/fontawesome/css/font-awesome.css

```diff
@@ -1,13 +1,13 @@
 /*!
- *  Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
+ *  Based on Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
```

Comment:
@stefanorigano I think we can consider that we fork font-awsome and leaving this in "static/lib" disturbs me. After a quick discussion with the JS team, I propose to move it in src/libs.